src/components/BackgroundImage/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface BackgroundImageProps {
+  children: React.ReactNode;
+  backgroundUrl?: string;
+  'data-test-id'?: string;
+}
+
+const BackgroundImage = ({ backgroundUrl, children, ...attributes }: BackgroundImageProps): JSX.Element => {
```

Comment:
let's export the `data-test-ids` here:

```js
export const backgroundImageTestIds = {
  base: 'background-image',
  children: 'background-image__children'
};
```

Then we can import them in the `index.test.tsx`  and test the component accordingly!